_posts/2019-11-06-machine-readable-output.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Machine-readable Output Formats
+author: José Iván López González
+layout: post
+piwik: true
+---
+
+Third party programs and scripts (e.g., YaST) parse the output of the *snapper* cli commands to get some information. For example, to find out the number of all pre and post snapshots, scripts can run the following:
+
+~~~
+snapper list | awk '/pre/||/post/{print $3}'
+~~~
+
+Output from *snapper list* command is a table intended to be read by humans but not scripts. This could make difficult to parse snapper outputs. But even worse, every change in this table output (e.g., by adding a new column or changing their values format) could ruin scripts that relied on the previous table version.
```

Comment:
* "The output"
* For me it sounds better: "to be read by humans, not by scripts". But it is maybe just a matter of preference.
* "this table ~~output~~".
* ~~by~~ adding a new column
* "their values format" -> "the format of their values"
* "on the previous ~~table~~ version" (the "table" word appears three times in this paragraph.